lib/Parser/Parse.cpp

```diff
@@ -7821,9 +7821,17 @@ ParseNodePtr Parser::ParseExpr(int oplMin,
                     }
                 }
             }
-            else if (nop == knopEllipsis && !fAllowEllipsis)
+            else if (nop == knopEllipsis )
             {
-                DeferOrEmitPotentialSpreadError(pnodeT);
+                if (!fAllowEllipsis)
+                {
+                    DeferOrEmitPotentialSpreadError(pnodeT);
+                }
+            }
+            else if (m_token.tk == tkExpo)
+            {
+                //Unary operator on the left handside of ** is unexpected, except ++, -- or ...
```

Comment:
@dilijev it works in ch with -es7exponentiationoperator specified.  Did you mean current stable release environments don't have that working?